cmd/agent/subcommands/run/command_windows.go

```diff
@@ -95,6 +96,7 @@ func StartAgentWithDefaults(ctxChan <-chan context.Context) (<-chan error, error
 			rcclient rcclient.Component,
 			forwarder defaultforwarder.Component,
 			logsAgent optional.Option[logsAgent.Component],
+			processAgent optional.Option[processAgent.Component],
```

Comment:
In windows, we should try to avoid having process agent component being injected at all.